docs/cedarling/cedarling-logs.md

```diff
@@ -32,9 +32,332 @@ There are three different log records produced by the Cedarling:
 
 ## Jans Lock Server
 
-In enterprise deployments, [Janssen Lock Server](../janssen-server/lock/) collects Cedarling 
-logs and can stream to a database or S3 bucket. The Cedarling decision logs provide compliance 
-evidence of usage of the domain's externalized policies. The logs are also useful for forensic 
+In enterprise deployments, [Janssen Lock Server](../janssen-server/lock/) collects Cedarling
+logs and can stream to a database or S3 bucket. The Cedarling decision logs provide compliance
+evidence of usage of the domain's externalized policies. The logs are also useful for forensic
 analysis to show everything the attacker attempted, both allowed and denied.
 
+## Example of log
 
+Json formatted is pretty prettified.
+Result of authorize is pretty bit because we log all `cedar-policy` entities information for forensic analysis and we can't cut data because it can be crucial.
```

Comment:
I believe the logs should not be prettified when stored. This approach reduces storage space requirements and minimizes bandwidth usage during transmission.

Since users are likely to view logs infrequently, they can prettify them on-demand when needed.

That said, I’m unsure if this PR includes the implementation for saving the logs, or if it’s just updating the documentation. I couldn’t find the relevant code changes for saving logs in this PR.

Reply:
It is prettified in this doc file. But in log is not. I will specify more clear

Reply:
Specified more clear message

Reply:
https://github.com/JanssenProject/jans/pull/10281/commits/2f0918d840b61c1da4901bd0b24d1deb6b600242

Reply:
okay that's good to know

Reply:
Logs can be stored in any format supported by serde. sparkv currently stores String instances. It might be worth generalising that to `Vec<u8>` or similar so that sparkv can store values serialised using binary formats.

Probably `KvEntry<T>` with T having some trait suitable for serde would work better.